systest/cluster/cluster.go

```diff
@@ -446,9 +499,9 @@ func (c *Cluster) reuse(cctx *testcontext.Context) error {
 	cctx.Log.Debugw(
 		"discovered cluster",
 		"bootnodes",
-		c.bootnodes,
+		len(c.BootNodes),
```

Comment:
I believe this should be

```suggestion
		c.bootNodesNum,
```

?

Reply:
Actually I wanted to get rid of the counter and use `len(c.BootNodes)` everywhere to avoid keeping the same information twice (and risking them going out of sync).

Reply:
🤷  I just found it inconsistent, a few lines further down there is also `c.smeshersNum` and not `len(c.Smeshers)`, but I'm not blocking because of this.

Reply:
I know, it is fixed now. Thanks for noticing.